### PR TITLE
Webhook events: add ability to parse it and missing PaymentIntentCanceled event type

### PR DESCRIPTION
**Pull request by lms11**
This PR makes it possible to:
1. Parse a Webhook Event, by adding the missing fields
2. Add missing PaymentIntentCanceled event type

### CODE REVIEW

**Comment by lms11** [comment]
Types are listed here: https://stripe.com/docs/api/events/types

**Comment by arlyon** [comment]
Thanks for this will review shortly :tada:

File: src/resources/webhook_events.rs [hunk 1]
```diff
@@ -205,11 +208,39 @@ pub enum EventType {
...
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct WebhookEvent {
+    /// Unique identifier for the object.
     pub id: EventId,
-    #[serde(rename = "type")]
-    pub event_type: EventType,
+
+    /// The connected account that originated the event.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub account: Option<Box<String>>,
```
Comment (arlyon) [inline src/resources/webhook_events.rs:216]: Is there a specific reason for boxing the string here?

File: src/resources/webhook_events.rs [hunk 2]
```diff
@@ -205,11 +208,39 @@ pub enum EventType {
...
+
+    /// The Stripe API version used to render `data`.
+    ///
+    /// *Note: This property is populated only for events on or after October 31, 2014*.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub api_version: Option<Box<String>>,
```
Comment (arlyon) [inline src/resources/webhook_events.rs:222]: See above

File: src/resources/webhook_events.rs [hunk 3]
```diff
@@ -205,11 +208,39 @@ pub enum EventType {
...
+
+    /// Has the value `true` if the object exists in live mode or the value `false` if the object exists in test mode.
+    pub livemode: bool,
+
+    /// Number of webhooks that have yet to be successfully delivered (i.e., to return a 20x response) to the URLs you've specified.
+    pub pending_webhooks: i64,
+
+    /// Information on the API request that instigated the event.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub request: Option<Box<NotificationEventRequest>>,
```
Comment (arlyon) [inline src/resources/webhook_events.rs:239]: See above

**Review by arlyon — CHANGES_REQUESTED** [review]
Looks good! Some small nitpicks but otherwise helpful. One thing I noticed is that apparently `tarpaulin` doesn't run against the examples (while `cargo test` does...) and this breaks one of them. I will get that sorted out in a later PR, but in the mean time could you please run `cargo test --features runtime-tokio-hyper` and fix the failing example? It's a trivial fix.

Thanks! :tada:

**Comment by arlyon** [comment]
Closed in favour of #233

**Pull request closed without merging**